### PR TITLE
Remove ONBOARDING_V2 flag for release

### PR DESCRIPTION
**Pull request by darkwing**
## Explanation

This PR removes the `ONBOARDING_V2` environment variable flag so we can release our new onboarding flow.

## Pre-merge author checklist

- [ ] I've clearly explained:
  - [ ] What problem this PR is solving
  - [ ] How this problem was solved
  - [ ] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

**Comment by danjm** [comment]
Let's make sure we have @PeterYinusa's, @seaona's and @kevinghim's approval on onboarding v2 before we merge this.

@darkwing could you do a walkthrough with Kevin to review the designs and features?

**Review by danjm — COMMENTED** [review]
code here looks good, but I didn't verify that every removed test case has an onboarding v2 equivalent. I'll try to check that later, or maybe other reviewers can

**Comment by jpuri** [comment]
A last occurrence [here](https://github.com/MetaMask/metamask-extension/blob/develop/ui/pages/home/home.component.js#L418)

File: ui/pages/routes/routes.component.js [hunk 1]
```diff
@@ -307,7 +305,7 @@ export default class Routes extends Component {
 
     const isInitializing = Boolean(
       matchPath(location.pathname, {
-        path: process.env.ONBOARDING_V2 ? ONBOARDING_ROUTE : INITIALIZE_ROUTE,
```
Comment (digiwand) [inline ui/pages/routes/routes.component.js:310]: I might be missing something... can we also remove the `INITIALIZE_ROUTE`?

Comment (darkwing) [reply ui/pages/routes/routes.component.js:310]: I'll do this in a separate PR, where I remove all the old components and routes.

**Review by brad-decker — APPROVED** [review]

**Review by pedronfigueiredo — APPROVED** [review]

**Pull request merged**